config.ini

```diff
@@ -1,8 +1,3 @@
-[DEFAULT]
-# Number of retries in case request fails. Applies globally
-# Can be changed on per-module basis
-http-retries = 5
-
```

Comment:
That was actually intentional, retries count can be specified in each module separately and changing it in `[main]` section does not affect other modules. 

https://github.com/hatkidchan/mastoposter/blob/d740ddb0b19dde73c72d0ae4b2888434cdb1f239/mastoposter/integrations/telegram.py#L109

Although I feel like moving it to `[main]` block would make more sense and it should propagate those parameters to other modules

Reply:
It fails with horrible error when in [DEFAULT]

Reply:
Never actually failed for me; though, retry feature was not tested thoroughly enough..